frontend/src/metabase/alert/alert.js

```diff
@@ -141,13 +141,23 @@ export const unsubscribeFromAlert = alert => {
   return async (dispatch, getState) => {
     await dispatch(unsubscribeFromAlertRequest.trigger(alert.id));
     dispatch({ type: UNSUBSCRIBE_FROM_ALERT });
-
-    // This delay lets us to show "You're unsubscribed" text in place of an
-    // alert list item for a while before removing the list item completely
-    setTimeout(
-      () =>
-        dispatch({ type: UNSUBSCRIBE_FROM_ALERT_CLEANUP, payload: alert.id }),
-      5000,
```

Comment:
this was bugging out due to mixing RTK + ye olde redux BS, so instead of doing a dumb little "you updated an alert" notification in the popover, i changed it to just trigger a notification, like every other alert update 🙄